downstream/modules/eda/proc-eda-copy-rulebook-activation.adoc

```diff
@@ -0,0 +1,33 @@
+[id="eda-copy-rulebook-activation"]
+
+= Copying a rulebook activation
+
+When setting up a new rulebook activation with field inputs that are similar to one of your existing rulebook activations, you can use the *Copy rulebook activation* feature instead of manually entering input into each field. While setting up rulebook activations can be a lengthy process, the ability to copy the required fields from an existing activation saves time and, in some cases, reduces the possibility of human error.
+
+.Procedure
+
+. On the Rulebook Activations page, click the *More Actions* icon *{MoreActionsIcon}* on the row of the activation you want to copy. The More Actions list is displayed with three options:
+** *Restart rulebook activation*
+** *Copy rulebook activation*
+** *Delete rulebook activation*
+. Select btn:[Copy rulebook activation]. 
++
+A message is displayed: "<Name of rulebook activation 1> copied." Initially, the newly copied activation is displayed as disabled on the Rulebook Activations page with the same name as the original activation followed by a time stamp in 24-hour format (for example, <Name of rulebook activation 1> @ 18:43:27).
++
+[IMPORTANT]
+====
+The original rulebook activation continues to run after you have copied it. If you try to enable the copied activation without editing the fields (including the Name field) to distinguish it from the original, a message is displayed reminding you that the rulebook activation was copied from an original, and enabling it might fail or result in duplicate jobs and other complications.
+====
+ 
+. Before you run the copied rulebook activation, edit the fields by completing the following: 
+.. Next to the copied rulebook activation, click the *Edit* icon. This takes you to the Edit form. 
+.. Edit the desired fields.
++
+[NOTE]
+====
```

Comment:
Suggestion: Can this be a step in the procedure, perhaps after step 'select copy rulebook activation' and the following note where it's mentioned the rulebook activation is created with the same name as that of the original one? It might get missed in the note. Your call though.

Reply:
The goal is to keep the edit task as simple as possible since we are creating an entire topic on editing activations. That's why we just say edit the fields. Just allowing the user to edit fields they desire, but making sure they know the significance of editing the Name field. SMEs didn't want there to be focus on the Name field only.

Reply:
Merging per @jself-sudoku 's request.